index.html

```diff
@@ -0,0 +1,165 @@
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5 ">
+                        <span class="ml-2">Data visualization</span>
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2">Challenges</span>
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2">Open Source Community</span>
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2">Gitter help rooms</span>
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2 ">videos</span>
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2">City Meetups</span>
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2">Wiki</span>
+                    </label>
+
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2">Forum</span>
+                    </label>
+        
+                    <label class="block items-center mt-3">
+                        <input type="checkbox" class="form-checkbox h-5 w-5">
+                        <span class="ml-2">Additional Courses</span>
+                    </label>
+                </div>
+              <div class="p-2 w-full">
+                <div class="relative">
+                  <label for="message" class="leading-7 ">Any comments or suggestions?</label>
+                  <textarea id="message" name="message" class="w-full bg-white rounded border border-gray-300 text-gray-600  h-32 text-base outline-none py-1 resize-none leading-6 px-3">Enter your comment here...</textarea>
+                </div>
+              </div>
+              <div class="p-2 w-full">
+                <button class="w-full bg-green-500 text-white border-0 py-2 px-8 rounded text-lg">Submit</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <h2> Thestart of a new begining</h2>
```

Comment:
- kindly consider giving a class to your header and use it to style your h2 which will help your project to meet the below requirement
- A .css file that includes one class with styles for your header (make it in your favorite color).

Reply:
{Optional] Please always try to give good reasons for requesting changes to explain to the student why the changes should be implemented , for example, tell the student why styling the header is important for his project and also remeber to always use emojis to show pleasantries or polite comments 😄